docs/sphinx/_toc.yml.in

```diff
@@ -1,24 +1,34 @@
+# Anywhere {branch} is used, the branch name will be substituted.
+# These comments will also be removed.
 defaults:
   numbered: False
-  maxdepth: 6
+  maxdepth: 7
 root: index
 subtrees:
-- entries:
-  - file: releasenotes
-  - file: citation
-  - file: install
-  - file: embed
-  - file: driver
-  - file: DebugAndLogging
-  - file: cache
-  - file: perfdatabase
-  - file: finddb
-  - file: find_and_immediate
-  - file: Getting_Started_FusionAPI
-  - file: MI200AlternateImplementation
-  - file: MIOpen_Porting_Guide
-  - file: apireference
-- caption: About
+- caption: What is MIOpen?
```

Comment:
this should not be a heading. it should be an intro page that describes the project.

Reply:
We are still discussing the content for the Intro page. For now, we have reorganized the ToC on what's available and accurate. We have also removed the reference to release notes, as it is obsolete.

Reply:
then we need to wait until we have a page to link to. It's confusing to have a heading represent a placeholder for a future page. A heading (='caption' in the toc) should only be there to represent pages that fit into that heading.

Reply:
you can use the README file to get a little general info for a 'What is ..?' page until the team gets a chance to add to it. this page doesn't need to super long or elaborate, just has to describe what the project is/does.